.github/workflows/pr-lint.yml

```diff
@@ -48,19 +51,29 @@ jobs:
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           report-json: ./eslint-report.json
+    
+      - name: Commit files
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
 
       - name: Prettier
         continue-on-error: true
         id: prettier
+        shell: bash
         run: |
-          ./prettier.sh
-          STATUS=$(git status --short output)
-          echo "::set-output name=STATUS::$STATUS"
-
-      - name: Commit prettier
-        uses: actions-js/push@master
-        if: steps.prettier.outputs.STATUS
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            if [[ $file == *.js || $file == *.ts || $file == *.jsx || $file == *.tsx || $file == *.mdx || $file == *.scss || $file == *.json ]]; then
+              echo "Run Prettier on $file"
+              npx prettier --config .prettierrc $file --write
```

Comment:
npx is super slow, please add it to the root package.json devdependencies

Reply:
Your rules! Will do!